_projects/enhance_coala_quickstart.md

```diff
@@ -67,7 +67,8 @@ However, there’s scope for improvement in the following areas:
   extracted from these files.
 * [X] Get the cEP https://github.com/coala/cEPs/issues/80 merged.
 
-##### [CODING PHASE 1: Extracting Information](https://gitlab.com/coala/GSoC/GSoC-2017/milestones/20) 30th May - 26th June
+##### [CODING PHASE 1: Extracting Information](https://gitlab.com/coala/GSoC/GSoC-2017/milestones/20) 30th May - 26th 
+       June
```

Comment:
see rendered version.  This doesnt work.

There is a longer section title in this page, so maybe this change wasnt necessary ?